sovrin_node/test/test_suspension.py

```diff
@@ -1,12 +1,11 @@
 import pytest
 
 from plenum.common.eventually import eventually
+from plenum.common.txn import TRUSTEE, TGB, STEWARD, TRUST_ANCHOR
```

Comment:
Discussed with Lovesh:
it was discussed with Jason sometime, and approved by him. An example why we need it: sending POOL_UPGRADE txn.